src/languages/en.js

```diff
@@ -612,14 +623,6 @@ export default {
             formLabel: 'View PDF',
         },
     },
-    pronouns: {
-        heHimHis: 'He/him',
-        sheHerHers: 'She/her',
-        theyThemTheirs: 'They/them',
-        zeHirHirs: 'Ze/hir',
-        selfSelect: 'Self-select',
```

Comment:
Note: Removing this for now, as we'll come back to "self-select" / pronoun customization later

Reply:
Convo here: https://expensify.slack.com/archives/C01GTK53T8Q/p1668116448072369?thread_ts=1666881832.702589&cid=C01GTK53T8Q